catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -125,6 +129,18 @@
   private static final Supplier<Boolean> ZERO_PAGESIZE_COMPATIBILTY =
       () -> Boolean.valueOf(System.getProperty(ZERO_PAGESIZE_COMPATIBILITY_PROPERTY));
 
+  @VisibleForTesting
+  static final String SOLR_COMMIT_NRT_METACARDTYPES = "solr.commit.nrt.metacardTypes";
+
+  private final Set<String> commitNrtMetacardType =
+      Sets.newHashSet(accessProperty(SOLR_COMMIT_NRT_METACARDTYPES, "").split(","));
+
+  @VisibleForTesting
+  static final String SOLR_COMMIT_NRT_COMMITWITHINMS = "solr.commit.nrt.commitWithinMs";
+
+  private final int commitNRtCommitWithinMs =
```

Comment:
:question: why not make it static?

Reply:
✏️ nit ...  It might be more optimal to use the `toInt()` method that accepts a default value instead of relying on the `getProperty()` one which returns a string that you are forced to parse after

Reply:
It probably would but this is a one time read.  It would also require me to overload `accessProperty`.

Reply:
it is a one time read every time a client is created right?

Reply:
Correct